### PR TITLE
Enable CONFIG_HIBERNATION in Mariner

### PR DESCRIPTION
**Pull request by BettyRain**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [X] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 

Enable CONFIG_HIBERNATION and required variables with it in kernel config

###### Change Log  

- Change kernel config files for x86 and arm64
- Change kernel release version & signatures

###### Does this affect the toolchain?  

**YES**

###### Associated issues  

- Fixes bug [#41144227](https://microsoft.visualstudio.com/OS/_workitems/edit/41144227/)

###### Test Methodology

- Pipeline build id: [258830](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=258886&view=results)

### CODE REVIEW

File: SPECS/kernel/kernel.spec [hunk 1]
```diff
@@ -406,6 +406,9 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %{_sysconfdir}/bash_completion.d/bpftool
 
 %changelog
+* Mon Oct 31 2022 Betty Lakes <bettylakes@microsoft.com> - 5.15.74.1-4
+- Turn on Configs for different TCP algorithms
```
Comment (neha170) [inline SPECS/kernel/kernel.spec:410]: Nit: changelog description

**Review by rlmenge — CHANGES_REQUESTED** [review]
Please open against main and not 2.0 branch

**Review by neha170 — COMMENTED** [review]
Could you check if the ARM pipeline built successfully as well, thanks

File: SPECS/bazel/bazel.spec [hunk 2]
```diff
@@ -49,6 +49,9 @@ cp ./scripts/packages/bazel.sh %{buildroot}/%{_bindir}/bazel
 %attr(0755,root,root) %{_bindir}/bazel-real
```
Comment (rlmenge) [inline SPECS/bazel/bazel.spec:50]: I don't think these auto upgrades were intended to be here?

Comment (neha170) [reply SPECS/bazel/bazel.spec:50]: I think they showed up due to the wrong target branch

**Review by Camelron — APPROVED** [review]
lgtm

**Comment by BettyRain** [comment]
> Could you check if the ARM pipeline built successfully as well, thanks

rerunning the arm64: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=259273&view=results

**Comment by christopherco** [comment]
Merged as part of https://github.com/microsoft/CBL-Mariner/pull/4369 . Closing this one

**Pull request closed without merging**